docs/source/user_guide/runtime-conf.md

```diff
@@ -101,36 +102,25 @@ elyra-metadata install runtimes \
        --cos_username=minio \
        --cos_password=minio123 \
        --cos_bucket=test-bucket \
-       --tags="['kfp', 'v1.0']" \
-       --schema_name=kfp
+       --tags="['kfp', 'v1.0']"
 ```
 
 Refer to the [Kubeflow Pipelines Configuration settings](#kubeflow-pipelines-configuration-settings) section for an explanation of the parameters.
 
 #### Modifying a runtime configuration
 
-To edit a runtime configuration:
+To edit a runtime configuration, use the `--replace` option along with `--name` and `--schema_name` (to locate the instance), followed by the modified property values.  In this case, we're updating the `api_password` and `tags` properties:
 
 ```bash
 elyra-metadata install runtimes \
        --replace \
        --name="my_kubeflow_pipelines_runtime" \
-       --display_name="My Kubeflow Pipelines Runtime" \
-       --api_endpoint=https://kubernetes-service.ibm.com/pipeline \
-       --auth_type="DEX_STATIC_PASSWORDS" \
-       --api_username=username@email.com \
+       --schema_name=kfp \
        --api_password=mynewpassword \
-       --engine=Argo \
-       --cos_endpoint=http://minio-service.kubeflow:9000 \
-       --cos_auth_type="USER_CREDENTIALS" \
-       --cos_username=minio \
-       --cos_password=minio123 \
-       --cos_bucket=test-bucket \
-       --tags="['kfp', 'v1.1']" \
-       --schema_name=kfp
+       --tags="['kfp', 'v1.1']"
```

Comment:
As is, the command fails:
```
$ elyra-metadata install runtimes \
       --replace \
       --name="my_kubeflow_pipelines_runtime" \
       --schema_name=kfp \
       --api_password=mynewpassword \
       --tags="['kfp', 'v1.1']"
[I 2022-01-19 12:44:38,517.517] Upgrading runtime kfp instance 'my_kubeflow_pipelines_runtime' to include runtime_type 'KUBEFLOW_PIPELINES'...
The following exception occurred saving metadata instance for schema 'kfp': Username and password are not supported for the selected Kubeflow authentication type.
```

Reply:
Thanks for catching this.  I can reproduce this and believe it is related to default value handling.  I need to take a closer look and submit a fix.

Reply:
This issue is all in the CLI processing where options are built from the schema and properties with default values that were NOT also specified on the command line during replace operations were being included in the update payload.  Fix (and test update) forthcoming.